doc/pachctl/pachctl.md

```diff
@@ -7,7 +7,7 @@
 
 Access the Pachyderm API.
 
-Envronment variables:
+Environment variables (and defaults):
```

Comment:
I seem to recall this error message was confusing to a user at one point. Do we want to consider changing it (which of course would also include adjusting the docs?)